streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -88,6 +88,58 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
     }
   }
 
+  test("SPARK-6766: batch info should be submitted") {
```

Comment:
How long does these two units tests take? I wonder whether it is best (in the interest of minimizing test run times) to collapse all these two unit tests into the earlier "batch info reporting".